fileManagment/FileTableViewCell.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  FileTableViewCell.swift
+//  fileManagment
+//
+//  Created by Rayan Taj on 23/11/2021.
+//
+
+import UIKit
+
+class FileTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var fileNameLabel: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
```

Comment:
do you remember the line in your `cellForRowAt` function?
```swift
fileCell.layer.cornerRadius = 15
```
you can remove it from there and replace it here inside `awakeFromNib` with:
```swift
self.layer.cornerRadius = 15
```